app/templates/end_of_application_feedback_page_4.html

```diff
@@ -40,7 +40,7 @@ <h1 class="govuk-fieldset__heading">
                         </div>
                         {% if form.errors.get('hours_spent') %}
                             <p id="hours-error" class="govuk-error-message">
-                                <span class="govuk-visually-hidden">Error:</span> Select a score
+                                <span class="govuk-visually-hidden">Error:</span> Enter number of hours
```

Comment:
Why aren't we using `form.errors.get('hours_spent')[0]` instead of hard coding the error message, like we do on line 11?